srcCs/perftest_publisher.cs

```diff
@@ -2340,8 +2337,7 @@ private void setTimeout(ulong executionTime)
         public const string _LatencyTopicName = "Latency";
         public const string _ThroughputTopicName = "Throughput";
         public const string _AnnouncementTopicName = "Announcement";
-        public DDS.Duration_t timeout_wait_for_ack =
-                DDS.Duration_t.DURATION_ZERO;
+        public const uint timeout_wait_for_ack_nanosec = 10000000;
```

Comment:
What about follow a standard between the languages and use the same name: 
timeout_wait_for_ack_nsec or timeout_wait_for_ack_nanosec

Reply:
Done